cmd/zc_traverser_blob.go

```diff
@@ -347,6 +347,15 @@ func (t *blobTraverser) parallelList(containerURL azblob.ContainerURL, container
 	}, t.tqueue, t.isSource, t.isSync, t.maxObjectIndexerSizeInGB)
 
 	for x := range cCrawled {
+
+		if x.EnqueueToTqueue() {
+			item, err := x.Item()
+			if err != nil {
+				panic("Error set for entry which needs to be inserted to tqueue")
+			}
+			t.tqueue <- item
+		}
+
```

Comment:
need to add the same logic to zc_traverser_file.go

add a comment explaining what we are doing here

//
// This is a special CrawlResult which signifies that we need to enqueue the given directory to tqueue for target traverser to process.
//

in all other traversers too

also, let's assert that when EnqueueToTqueue() is true, the item is indeed a directory.
can we do it?